server/test/serverTest.js

```diff
@@ -556,7 +556,7 @@ describe('Chatbot server', () => {
     })
 
     it('should send the initial text message after a valid single click request to /flic_button_press', async () => {
-      // eslint-disable-next-line prettier/prettier
+      // prettier-ignore
```

Comment:
I'm curious about this change because the reason I did it using the `eslint-disable` is that `prettier-ignore` seems to affect chunks of formatting without a way to reliably target a single line, when I had used it it got rid of a bunch of formatting complaints in the same area that hadn't been fixed yet and should have been instead of just this `await` chain

Reply:
Ooo, good to know. I'll have to look into it more. I switched it because my VSCode extension was still formatting this line even though it had the eslint ignore. That's probably because I have the prettier plugin turned on but not the eslint one (I find it's underlines while I'm still typing really annoying). I can switch it back.

Reply:
oooh weird! Maybe mine is extension-related too then but I think I have both on 🤔

Reply:
I switched it back. Although I thiiiiiink the prettier ignore would have worked too, according to its docs (https://prettier.io/docs/en/ignore.html#javascript), but I'm happy with the eslint ignore as long as I know that that's what's happening.